### DISCUSSION TITLE
Has the long video demo been removed?

### DISCUSSION THREAD

**Discussion by SeanBackstrom** [open-mmlab/mmaction2#2556] [General]
I have been using the base code for long_video_demo for awhile. I'm curious for what reason it has been removed. Was there some sort of problem with it? Thank you so much for taking the time to answer.

**Comment by cir7** [comment]
It's just that due to insufficient human resources, we didn't have time to migrate some demos, welcome all kinds of contributions to us

**Reply by SeanBackstrom** [reply]
Thank you very much for the response. I have used MMlab, (MMDet and MMAction2) for years in production now, I would love to try to contribute to something that has been so foundational for me

**Reply by cir7** [reply]
Thanks for your contribution. Please feel free to contact us if you have any questions
